extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -132,7 +132,9 @@ public static JavaSettings getInstance() {
                 getStringValue(host, "default-http-exception-type"),
                 getBooleanValue(host, "use-default-http-status-code-to-exception-type-mapping", false),
                 host.getValue(new TypeReference<Map<Integer, String>>() {}.getType(),
-                    "http-status-code-to-exception-type-mapping"));
+                    "http-status-code-to-exception-type-mapping"),
+                getBooleanValue(host, "handle-partial-update", false)
```

Comment:
nit: I think we can drop `handle` from the name. Just `partial-update` should be sufficient.

Reply:
updated